content/rc/cloud-accounts/iam-resources/terraform.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: Create IAM resources using Terraform
+linkTitle: Terraform
+Weight: $weight
+categories: ["RC"]
+alwaysopen: false
+aliases: /rc/how-to/creating-iam-resources/terraform/
+         /rc/how-to/creating-iam-resources/terraform.md
+         /rc/cloud-accounts/iam-resources/terraform/
+         /rc/cloud-accounts/iam-resources/terraform.md
+
+---
+You can use [HashiCorp Terraform](https://www.terraform.io/intro/index.html) to create identity and access (IAM) resources to support AWS cloud account access to Redis Enterprise Cloud subscriptions.
+
+The following example uses the `terraform-aws-Redislabs-Cloud-Account-IAM-Resources` module, located in Amazon%nbsp;S3:
```

Comment:
Should "%nbsp;" be changed to "&nbsp;"? It looks like the current tag doesn't work on the staged preview.

Reply:
Actually, it's supposed to be "&nbsp;" (which is an HTML entity for a non-breaking space). Here, it keeps the two words from breaking across lines.

Fixed.